_data/communication.yml

```diff
@@ -1,4 +1,11 @@
 websites:
+    - name: Action Network
+      url: https://actionnetwork.org/
+      img: actionnetwork.png
+      tfa: Yes
+      software: Yes
+      doc: https://help.actionnetwork.org/hc/en-us/articles/217206826
```

Comment:
The site does not appear to have any public documentation regarding their implementation of 2-factor authentication. The `doc` link only prompts the user to sign in to "[enroll in 2-factor authentication](https://help.actionnetwork.org/hc/en-us/articles/203112329-Edit-your-profile-avatar-email-address-name-password-or-homepage)..." 

@j-ro, would you be able to attach a screenshot that shows how the site has implemented software 2FA?